package.json

```diff
@@ -39,10 +39,10 @@
     "eslint-import-resolver-babel-module": "^5.0.1",
     "eslint-plugin-import": "^2.16.0",
     "eslint-plugin-prettier": "^3.1.4",
+    "husky": "^4.2.5",
```

Comment:
What are the advantages of `husky`?

Reply:
In my researches, I found that it's more often used and have some additional hooks like `pre-push`. [Docs](https://github.com/typicode/husky)

Reply:
Nice.